Tests/SwerverTests/HttpRequestParserSpec.swift

```diff
@@ -0,0 +1,24 @@
+import Quick
```

Comment:
Did you TDD this code? This is just a gut reaction here, but it looks like your test coverage is pretty low compared to the amount of code you wrote.

Reply:
I spent a lot of time worried about code coverage because I didn't write any acceptance tests to test socket connection which probably made up the majority of the entire code base. other then that I believe I have covered all the others modules such as request handler, parser, router etc and tested their behaviors rather then lines of codes.